core/src/main/java/org/kiwix/kiwixmobile/core/dao/entities/NotesEntity.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2020 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.kiwix.kiwixmobile.core.dao.entities
+
+import io.objectbox.annotation.Entity
+import io.objectbox.annotation.Id
+import org.kiwix.kiwixmobile.core.page.notes.adapter.NoteListItem
+
+@Entity
+data class NotesEntity(
+  @Id var id: Long = 0L,
+  val zimId: String,
+  var zimFilePath: String?,
+  val zimUrl: String,
+  // @Unique(onConflict = ConflictStrategy.REPLACE) add resolve conflict dependency update required
```

Comment:
it's supported by the object box, but a dependency update is required.

Reply:
@4shutosh We updated the dependency recently. Please rebase the commits.